client-runtime/src/main/java/com/microsoft/rest/interceptors/BaseUrlHandler.java

```diff
@@ -1,53 +0,0 @@
-/**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- */
-
-package com.microsoft.rest.interceptors;
-
-import okhttp3.HttpUrl;
-import okhttp3.Interceptor;
-import okhttp3.Request;
-import okhttp3.Response;
-
-import java.io.IOException;
-
-/**
- * Handles dynamic replacements on base URL. The arguments must be in pairs
- * with the string in raw URL to replace as replacements[i] and the dynamic
- * part as replacements[i+1]. E.g. {subdomain}.microsoft.com can be set
- * dynamically by setting header x-ms-parameterized-host: "{subdomain}, azure"
- */
-public final class BaseUrlHandler implements Interceptor {
```

Comment:
This is now handled by RestProxy, right?

Reply:
Correct.